backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/dto/StockDto.java

```diff
@@ -46,6 +47,7 @@ public abstract class StockDto implements Serializable {
 
     private UUID uuid;
 
+    @Valid
```

Comment:
When using the frontend to update a material stock the following error is thrown:

```
jakarta.validation.UnexpectedTypeException: HV000030: No validator could be found for constraint 'jakarta.validation.constraints.Pattern' validating type 'java.lang.Boolean'. Check configuration for 'material.materialFlag'
```

I checked the MaterialDto. For some reason we added a string annotation to the boolean material - which I think is by accident. After removing it, my freehand testing in Frontend + integration test works :)

Please fix.

Reply:
Done, the error was caused by a string pattern matcher that was not suitable for a boolean attribute.